toolchest_client/tools/api.py

```diff
@@ -504,6 +505,72 @@ def humann3(inputs, output_path=None, tool_args="", mode=HUMAnN3Mode.HUMANN,
     return output
 
 
+def jupyter(notebook, inputs=None, output_path=None, requirements=None,
+            docker_tag="latest", grace_period_seconds=None, port=None,  **kwargs):
+    """Get a spawn token for a Jamsocket Jupyter notebook environment via toolchest.
+
+    :param notebook: path to the Jupyter Notebook to run on environment start.
+    :param inputs: (optional) path(s) to the files that will be accessible in the spawned environment. Additional
+    Jupyter Notebooks can be provided here.
+    :param requirements: (optional) path to a pip requirements.txt file to install dependencies for the notebook
+    environement.
+    :param output_path: (optional) local path to where the output file(s) will be downloaded.
+    :param docker_tag: (optional) additional arguments to be passed to your script as command line arguements.
```

Comment:
typo: arguments

Reply:
The whole message is actually wrong as it's for `tool_args`and not related to docker tags. I'll update that to be for docker tags but will not fix the typo in this pr since there's 32 instances of it in `api.py`.